cli/src/pcluster/models/imagebuilder.py

```diff
@@ -99,6 +121,10 @@ def get_source_config(self):
         except (AWSClientError, KeyError):
             return None
 
+    def update_stack(self):
```

Comment:
update_stack_info?

because update_stack let me think that we're updating the stack in cfn.

Reply:
Done.